cli/app/ui_terminal_cursor.go

```
@@ -0,0 +1,282 @@
+}
+
+func terminalCursorControlSequenceInvalidatesPlacement(sequence string, parser *xansi.Parser) bool {
+	if sequence == "\r" || sequence == "\n" {
+		return true
+	}
+	command := xansi.Cmd(parser.Command())
+	switch command.Final() {
+	case 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'J', 'f':
+		return true
+	case 'h', 'l':
+		return sequence == xansi.SetAltScreenSaveCursorMode || sequence == xansi.ResetAltScreenSaveCursorMode
```

golangci-lint (2.11.4)

[error] SA1019: xansi.SetAltScreenSaveCursorMode is deprecated: use [ModeAltScreenSaveCursor] instead. 

(staticcheck)